modules/compass.md

```diff
@@ -0,0 +1,80 @@
+# Overview
+
+```
+Module Name: Compass Bidder Adapter
+Module Type: Compass Bidder Adapter
+Maintainer: sa-support@brightcom.com
+```
+
+# Description
```

Comment:
Add another # to this title.

## Description instead of # Description.

This should clear up another Codacy issue.